src/plonetheme/eeq/models/office.xml

```diff
@@ -24,5 +24,12 @@
       <title>LinkedIn url</title>
       <required>False</required>
     </field>
+    <field name="navigation_links" type="z3c.relationfield.schema.RelationList">
+      <title>Links on the right of the top bar</title>
+      <required>False</required>
+      <value_type type="z3c.relationfield.schema.RelationChoice">
```

Comment:
Generally I avoid using `RelationChoice` because I think the implementation is kind of a disaster, but it's probably fine here. Generally I prefer to use a List field with TextLine values (for UUIDs) and apply the RelatedItemsWidget to that, which generally works without additional configuration and doesn't spin up the super-complex RelationCatalog, IntiId, KeyReference machinery.

Reply:
Noted.